pynest/nest/tests/test_topology_gamma_kernel.py

```diff
@@ -0,0 +1,124 @@
+# -*- coding: utf-8 -*-
+#
+# test_topology_gamma_kernel.py
+#
+# This file is part of NEST.
+#
+# Copyright (C) 2004 The NEST Initiative
+#
+# NEST is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 2 of the License, or
+# (at your option) any later version.
+#
+# NEST is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with NEST.  If not, see <http://www.gnu.org/licenses/>.
+
+"""
+Test of events
+"""
+
+import numpy as np
+import scipy.stats as sts
+import unittest
+import nest
+import nest.topology as tp
+from collections import Counter
+
+
+@nest.check_stack
+class TopologyGammaKernelTestCase(unittest.TestCase):
+    """Tests of gamma kernel in topology"""
+
+    def test_targets(self):
+        """Object targets"""
+
+        nest.ResetKernel()
+
+        extent = [100., 100.]
+        npop = 5000
+        positions = np.random.uniform(-extent[0] / 2, extent[1] / 2, [npop, 2])
+        ncon = 10000
+        kappa = 4.
+        theta = 4.
+        radius = extent[0] / 2 * np.sqrt(2)
+        dx = .5
+        x = np.arange(0, radius + dx, dx)
+
+        ly1 = tp.CreateLayer({
+            'positions': [[0., 0.]],
+            'elements': 'iaf_psc_alpha',
+            'extent': extent,
+            'edge_wrap': True
+        })
+
+        ly2 = tp.CreateLayer({
+            'positions': positions.tolist(),
+            'elements': 'iaf_psc_alpha',
+            'extent': extent,
+            'edge_wrap': True
+        })
+
+        tp.ConnectLayers(ly1, ly2, projections={
+            'connection_type': 'divergent',
+            'allow_oversized_mask': True,
+            'mask': {
+                'circular': {
+                    'radius': radius
+                }
+            },
+            'kernel': {
+                'gamma': {
+                    'kappa': kappa,
+                    'theta': theta
+                }
+            },
+            'number_of_connections': ncon,
+            'allow_multapses': True,
+            'allow_autapses': True,
+        })
+
+        gamma1 = sts.gamma.pdf(x, a=kappa, scale=theta)
+        gamma_normed1 = gamma1 / float(np.sum(gamma1)) / dx
```

Comment:
As these are only used when plotting for debugging, they should be moved down to the debug block, or removed.